UI/OptionsWnd.cpp

```diff
@@ -489,65 +489,66 @@ OptionsWnd::OptionsWnd():
     GG::Connect(show_font_texture_button->LeftClickedSignal, &ShowFontTextureWnd);
 
     CreateSectionHeader(current_page, 0, UserString("OPTIONS_FONT_SIZES"));
-    IntOption(current_page, 0, "UI.font-size",       UserString("OPTIONS_FONT_TEXT"));
```

Comment:
Completely unrelated to the PR itself and it's by no means a request to change this now, but it maybe better to explain WHY I don't want to see such changes in PRs:  It took literally 10 reads for me until I realized that you've added a new OptionsDB entry.

 This whole ' tabularizing' entries is just some horrible habit we should ban for this very reason from our code base, it makes changes much harder to understand. As I said this is not against your change specifically @dbenage-cx.